www/docs/en/6.x/config_ref/images.md

```diff
@@ -135,6 +136,22 @@ different screen resolutions.
         <icon src="res/Windows/Wide310x150Logo.scale-240.png" width="744" height="360" />
     </platform>
 ```
+
+For Windows it's also possible to specify icons destination using `target` attribute. This approach has the following advantages:
```

Comment:
Because of these advantages should this be the recommended way - and hence, be the first example?